core/src/plugins/filed/python/ovirt/BareosFdPluginOvirt.py

```diff
@@ -107,11 +118,20 @@ def check_options(self, mandatory_options=None):
         """
         Check Plugin options
         Note: this is called by parent class parse_plugin_definition().
-        This plugin does not yet implement any checks.
+        This plugin does not yet implement any checks for plugin options,
+        but it's used to report if the Python oVirt SDK is not installed.
         If it is required to know if it is a backup or a restore, it
         may make more sense to invoke the options checking from
         start_backup_job() and start_restore_job()
         """
+
+        if SDK_IMPORT_ERROR:
+            bareosfd.JobMessage(
+                bareosfd.M_FATAL,
+                "Please install the Python SDK for oVirt Engine API.\n",
```

Comment:
That message doesn't tell me what package to get from pip. Maybe we should mention `pyvmomi` here?